src/sentry/models/organizationcontributors.py

```diff
@@ -21,7 +21,7 @@ class OrganizationContributors(DefaultFieldsModel):
 
     organization = FlexibleForeignKey("sentry.Organization", on_delete=models.CASCADE)
 
-    integration_id = HybridCloudForeignKey("sentry.Integration", on_delete="CASCADE")
+    integration_id = HybridCloudForeignKey("sentry.Integration", on_delete="SET_NULL", null=True)
```

Comment:
I just want to check that we do want to leave the id here and just set it to DO NOTHING. Not sure if it's useful as a historical thing or not, just wanted to check